Assets/00_Content/Scripts/Vikings/States/SatisfiedVikingState.cs

```diff
@@ -39,6 +40,8 @@ public class SatisfiedVikingState : VikingState {
 			if (givenItem != null) {
 				if (givenItem is Tankard tankard)
 					tankard.IsFull = false;
+				if (givenItem is Food food)
+					return;
```

Comment:
Perhaps add another property on desires to control if they will be thrown by the viking instead of manually adding it here?

Reply:
Agreed!